### PR TITLE
new record, small additions

### PR DESCRIPTION
**Pull request by thea-m**
I could not find the small introduction in LIT6024IntroMM anywhere, so I created a new record for this in combination with the folllowing hymn...but maybe you know it?

### CODE REVIEW

File: 4001-5000/LIT4615Salam.xml [hunk 1]
```diff
@@ -50,17 +50,32 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
                         ለመስቀልከ፡ እግዚኦ፡ ንፌኑ፡ ስብሐተ፡ ሃሌሉያ፡ ሰላም፡ ለከ፡ ዮሐንስ፡ ወልደ፡ ነጐድጓድ፨ 
                         ሰላም፡ ለከ፡ ዮሐንስ፡ አቡ፡ ቀለምሲስ፨ ሰላም፡ ለከ፡ ዮሐንስ፡ ቴዎጎሎስ፨
                     </ab>
                 </div>
+                
+                <div type="textpart" subtype="explicit" xml:lang="gez">
+                    <note xml:lang="en">The following explicit is taken from <ref type="mss" corresp="BDLaethd20"/></note>
+                    <ab>ወላዲተ፡ አምላክ፡ ማርያም፡ እንበለ፡ ሰብሳብ፡ ወሩካቤ፡ በል፨
+                        ወእምዝ፡ ሰላም፡ ለኪ፡ ማርያም፡ ኆኅተ፡ ምሥራቅ፡ ዘወልደ፡ ኖሬ፡ ሰላም፡ ለኪ፡ ዓፀድ፡ ዘኤስኬድሬ፨
+                        ሰላም፡ ለኪ፡ ሥርጉት፡ በወራውሬ፨ ሰ፡ ለ፡ እንተ፡ ወለድኪ፡ ፈጣሬ፨
```
Comment (MassimoVilla) [inline 4001-5000/LIT4615Salam.xml:75]: We may state that ሰ፡ ለ፡ is the abbreviated form of ሰላም፡ ለኪ፡ (e.g. through `<expan><abbr></abbr><ex></ex>`?)

File: new/LIT6024IntroMM.xml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition">
+                <note>The following excerpts are taken from <ref type="mss" corresp="BDLaethd20"/></note>
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <ab>በፈቃደ፡ አቡሁ፡ ወበሥምረተ፡ ርእሱ፡ አመ፡ ርእሶ፡ ይሰቅል፨
+                        ወአመ፡ ተዓገሠ፡ ኵሎ፡ ሐፃውንተ፡ ቅዱስ፡ መስቀል፨ ሶበ፡ ሰማዕኪ፡
+                        ማርያም፡ ዘነገሩኪ፡ በቃል። ዘውኅዘ፡ እምአዕይንትኪ፡ ከመ፡ ውኂዘ፡ ማይ፡ ዘይፈለፍል፨
```
Comment (MassimoVilla) [inline new/LIT6024IntroMM.xml:60]: Perhaps እምአዕይን**ቲ**ኪ፡? with -_i_- between pl. form and suffix

File: new/LIT6024IntroMM.xml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+                            ለመስቀልከ፡ እግዚኦ፡ ንፌኑ፡ ስብሐተ፨
+                            ሃሌ፡ ሉያ፡ ሃሌ፡ ሉያ፡ ሃሌ፡ ሉያ፡ 
+                            ሰላም፡ ለከ፡ ዮሐንስ፡ ወልደ፡ ነጐድጓድ፨
+                        </ab>
+                    </div>
+                    
+                    <div type="textpart" subtype="explicit">
+                        <ab>ወላዲተ፡ አምላክ፡ ማርያም፡ እንበለ፡ ሰብሳብ፡ ወሩካቤ፡ በል፨
+                            ወእምዝ፡ ሰላም፡ ለኪ፡ ማርያም፡ ኆኅተ፡ ምሥራቅ፡ ዘወልደ፡ ኖሬ፡ ሰላም፡ ለኪ፡ ዓፀድ፡ ዘኤስኬድሬ፨
+                            ሰላም፡ ለኪ፡ ሥርጉት፡ በወራውሬ፨ ሰ፡ ለ፡ እንተ፡ ወለድኪ፡ ፈጣሬ፨
```
Comment (MassimoVilla) [inline new/LIT6024IntroMM.xml:77]: I think we should encode the entire text ሰላም፡ ለኪ፡ (the abbreviation is a scribal habit of the single text witness) or state somehow that this is an abbreviation (`<expan><abbr></abbr><ex></ex>`?)

Comment (thea-m) [reply new/LIT6024IntroMM.xml:77]: @PietroLiuzzo can `<abbr>` and `<expan>` be used within the text (I suppose so, but this hasn't yet be documentedly done in BM)? In the TEI guidelines examples, both are contained by `<choice>`.

**Comment by thea-m** [comment]
Thank you!

File: new/LIT6024IntroMM.xml (hunk 3 above)
Comment (thea-m) [reply new/LIT6024IntroMM.xml]: @PietroLiuzzo

Comment (PietroLiuzzo) [reply new/LIT6024IntroMM.xml]: yes @thea-m, @MassimoVilla  is right ([gl page with example](https://betamasaheft.eu/Guidelines/?id=text-encoding)),  no need for `<choice>`

**Comment by thea-m** [comment]
I think that this can be merged?

**Pull request merged**